.github/workflows/release.yaml

```diff
@@ -1,21 +1,52 @@
 on:
   release:
     types: [created]
+  push:
+    tags:
+      - "*"
 name: Handle Release
```

Comment:
@Ehco1996 i meant actually *only* adding this trigger to the other workflow file and removing the content below. The reason is that the sections below are mostly duplicated code and now we always have to keep two CI files in sync. Whenever we change the image tagging code in ci.yaml we also need to update release.yaml, which is easy to forget and is likely going to cause issues. I would suggest we make the addition only in ci.yaml 👍